src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -2,15 +2,19 @@
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 
 @Entity
 @Table(name = "PRODUCT")
 public class Product {
+  
     private Long id;
     private String name;
     private String description;
     private double price;
     private String groupId;
+    private Group group;
```

Comment:
Pole `group` nie jest potrzebne jeśli posiadamy `groupId` - o tym koncepcie (przypisywanie id zamiast całego obiektu) rozmawialiśmy w trakcie ostatniej rozmowy. Możemy temat poruszyć podobnie, było z tym kilka problemów

Reply:
Jestem za tym aby przypisywać cały obiekt - jeżeli ma być String to prośba o pomoc, jak to ma wyglądać.